src/qt/transactionview.cpp

```diff
@@ -112,6 +113,17 @@ TransactionView::TransactionView(const PlatformStyle *platformStyle, QWidget *pa
     amountWidget->setValidator(new QDoubleValidator(0, 1e20, 8, this));
     hlayout->addWidget(amountWidget);
 
+    /* Delay before before filtering transactions in ms */
+    static const int inputFilterDelay = 200;
```

Comment:
Use snake case. Comment to indicate unit?

FWIW there is this [global constant](https://github.com/bitcoin/bitcoin/blame/master/src/qt/guiconstants.h#L9) cc @sipa.